get_library.sh

```diff
@@ -105,6 +106,9 @@ elif [ $1 == 'uuid' ]; then
 elif [ $1 == 'yaml-cpp' ]; then
     URL=https://github.com/creativa77/yaml-cpp.git
     COMP='git'
+elif [ $1 == 'rospkg' ]; then
+    URL=https://github.com/ros-infrastructure/rospkg.git
```

Comment:
Should we rely on the master? Wouldn't it be better to use a specific commit so we don't depend on external changes?